index.js

```diff
@@ -198,9 +198,24 @@ var publicAgent = {
   private_: function() { return agent; }
 };
 
-module.exports = global._google_trace_agent = publicAgent;
+/**
+ * @param {object} options - These are currently ignored
```

Comment:
Could we add a todo here so the comment can be updated when we refactor configuration.